hsmd/libhsmd.c

```diff
@@ -2021,6 +2065,8 @@ u8 *hsmd_init(struct secret hsm_secret,
 		WIRE_HSMD_SIGN_ANCHORSPEND,
 		WIRE_HSMD_SIGN_HTLC_TX_MINGLE,
 		WIRE_HSMD_SIGN_SPLICE_TX,
+		WIRE_HSMD_CHECK_OUTPOINT,
```

Comment:
I think we only want one capability for both messages.  we should have signers implement one but not the other.

Reply:
But `check_outpoint` is useful for other things than splices, no?

Reply:
Agree

Reply:
I agree, makes sense!

Done :)